.github/workflows/test.yml

```diff
@@ -2,17 +2,51 @@ name: 'Test setup-gazebo'
 on:
   workflow_dispatch:
```

Comment:
Instead of trigger this manually, I would prefer to run it on PR and other pushed to the repo. And even a cron daily to check if everything works as expected. We can use the ROS approach to this:
```yaml
on:
  pull_request:
  push:
    branches:
      - master
      - "releases/*"
  schedule:
    # Run the CI automatically twice per day to look for flakyness.
    - cron: "0 */12 * * *"
```